pkg/reading/issue.go

```diff
@@ -0,0 +1,6 @@
+package reading
+
+type Issue struct {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [exported type Issue should have comment or be unexported](https://app.codacy.com/app/TurnsCoffeeIntoScripts/jira-api-resource/pullRequest?prid=4570750)